### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by deathbaba**

### CODE REVIEW

File: src/FileStorageQueue/exponential_retry_strategy.h [hunk 1]
```diff
@@ -10,6 +10,15 @@
 #include "exception.h"
 #include "../Bricks/time/chrono.h"
 
+#ifdef _MSC_VER
```
Comment (dkorolev) [inline src/FileStorageQueue/exponential_retry_strategy.h]: These would be best to move to ```port.h```.

File: src/FileStorageQueue/fsq.h [hunk 2]
```diff
@@ -317,7 +317,7 @@ class FSQ final : public CONFIG::T_FILE_NAMING_STRATEGY,
     // Step 1/4: Get the list of finalized files.
     typedef std::vector<FileInfo<T_TIMESTAMP>> FileInfoVector;
     const FileInfoVector& finalized_files_on_disk = ScanDir([this](const std::string& s, T_TIMESTAMP* t) {
-      return T_FILE_NAMING_STRATEGY::finalized.ParseFileName(s, t);
+      return this->finalized.ParseFileName(s, t);
```
Comment (dkorolev) [inline src/FileStorageQueue/fsq.h]: Does Windows not compile this?

Comment (deathbaba) [inline src/FileStorageQueue/fsq.h]: No

File: src/FileStorageQueue/exponential_retry_strategy.h (hunk 1 above)
Comment (deathbaba) [inline src/FileStorageQueue/exponential_retry_strategy.h]: This repo should compile in our production now, you can fix cauldrons later.

**Pull request merged**